healthtools/scrapers/base_scraper.py

```diff
@@ -65,6 +65,7 @@ def scrape_site(self):
                 delete_batch.extend(delete_docs)
             except Exception as err:
                 skipped_pages += 1
+                self.post_to_slack("ERROR: scrape_site() - source: {} - page: {} - {}".format(url, page_num, err))
                 print "ERROR: scrape_site() - source: {} - page: {} - {}".format(url, page_num, err)
```

Comment:
Considering the error message is the same printed and to Slack, should we not instead use an "error method" that sends the relevant variables?